newerhoods/setup.R

```diff
@@ -1,21 +1,24 @@
+require(devtools)
```

Comment:
@kaushik12 I made some changes to the install dependencies script here. Let me know if they don't make sense or if they mess up the standard install process. I had to use a specific version of "rgeos" and install lattice manually to get it to install on linux